app.py

```diff
@@ -81,24 +80,22 @@ async def comment_cleaner():
                 issue_id = comment["issue_id"]
                 comment = await TicketFeedbackHandler().deleteComment(owner, repo, comment_id)
                 print(f"Print Delete Task,{comment}", file=sys.stderr)
-                print(SupabaseInterface().deleteComment(issue_id))
+                print(SupabaseInterface.get_instance().deleteComment(issue_id))
 
 async def fetch_github_issues_from_repo(owner, repo):
-    url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+    try:
+        response = await GithubAdapter.fetch_github_issues_from_repo(owner,repo)
+        if response:
+            return response
+        else:
+            print(f"Failed to get issues: {response}")
+                
+    except Exception as e:
+        logger.info(e)
+        pass
```

Comment:
why is 
```python
pass
```

here?

Reply:
Im not sure to raise a exception in this place ot not so i used pass here @KDwevedi